### PR TITLE
Subscription example

### PR DESCRIPTION
**Pull request by angrykoala**
# Description
An example for local subscriptions

### CODE REVIEW

File: examples/subscriptions/typedefs.graphql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type Movie {
+    title: String!
+    released:Int
```
Comment (darrellwarde) [inline examples/subscriptions/typedefs.graphql:3]: Sorry for being pedantic, this hurts my life. 😆 

```suggestion
    released: Int
```

File: examples/subscriptions/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+    "name": "neo-place",
```
Comment (darrellwarde) [inline examples/subscriptions/package.json:2]: Needs a new name!

File: examples/subscriptions/database-setup.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+};
+
+function getCanvasStatement(size, color = "#FFFFFF") {
+    const upperIndex = size - 1;
+
+    return `UNWIND range(0, ${upperIndex}) AS x
+    UNWIND range(0, ${upperIndex}) AS y
+    MERGE(p:Pixel { position: [x, y]})
+    ON CREATE SET p.color="${color}"`;
+}
```
Comment (darrellwarde) [inline examples/subscriptions/database-setup.js:17]: Copy and paste?

File: examples/subscriptions/create-plugin.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        constructor() {
+            this.events = new EventEmitter();
+        }
+
+        publish(eventMeta) {
+            this.events.emit(eventMeta.event, eventMeta);
+        }
+    }
+
+    return new SubscriptionsPlugin();
```
Comment (darrellwarde) [inline examples/subscriptions/create-plugin.js:14]: We can use the single instance one now right?

File: examples/subscriptions/server.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    // Setup websocket server on top of express http server
+    const wsServer = new WebSocketServer({
+        server: httpServer,
+        path: "/graphql",
+    });
+
+    // Build Neo4j/graphql schema
+    const schema = await neoSchema.getSchema();
+
+    await neoSchema.assertIndexesAndConstraints({ options: { create: true } });
```
Comment (darrellwarde) [inline examples/subscriptions/server.js:54]: Probably not necessary in this example?

**Review by darrellwarde — CHANGES_REQUESTED** [review]
Broadly looks good! Currently included the neo place work, so let's merge that first. A few suspected copy/paste errors.

**Comment by angrykoala** [comment]
Stale PR, neo/place was already merged

**Pull request closed without merging**